win32drv/win32drv.c

```diff
@@ -182,6 +186,17 @@ static WPARAM volatile g_keyboard_value = 0;
  *   GLOBAL FUNCTIONS
  **********************/
 
+EXTERN_C void lv_win32_add_all_input_devices_to_group(
+    lv_group_t* group)
+{
+    if (group)
```

Comment:
Just nitpicking: It seems like a possible use case to not assign any groups (set `NULL`)  on all input devices.  What do you think?

Regardless of this, I like this new approach.

Reply:
I have do some upgrade and hope it will be OK.